mne/decoding/tests/test_csp.py

```diff
@@ -23,6 +23,28 @@
 start, stop = 0, 8
 
 
+def simulate_data(target, n_trials=100, n_channels=3, random_state=42):
+    """Simulate data according to an instantaneous mixin model.
+
+    Data are simulated in the statistical source space, where one source is
+    modulated according to a target variable, before being mixed with a
+    random mixing matrix.
+    """
+    rs = np.random.RandomState(random_state)
+
+    # generate a orthogonal mixin matrix
+    mixing_mat = rs.randn(n_channels, n_channels)
+    _, mixing_mat = np.linalg.eigh(np.dot(mixing_mat.T, mixing_mat))
```

Comment:
Due to PCA/SVD equivalence I think this is equivalent to `mixing_mat = linalg.svd(rs.randn(n_channels, n_channels))[0]` if you want to save a line.